### PR TITLE
Feature/js survey styling

### PR DESCRIPTION
**Pull request by jordy1611**
### Type of change made:
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Testing - building tests
- [x] Formatting - no new features
- [x] Styling- no new features
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description
* Styled a sample survey page.
* Displays 6 categories each with 6 images.

### How should this be manually tested?
* I would just navigate to the survey page to make sure it all loads correctly.

### Any background context you want to provide?
* Missing the button at the end of the survey. I think a pre-styled button already exists.
* This is not connected to store

### Checklist
- [x] I have tested this code
- [ ] I have updated the Readme

### Questions:
What colors look better the grey? Was thinking the default purple.

### CODE REVIEW

File: code/web/src/modules/styleSurvey/CategoryImage.js [hunk 1]
```diff
@@ -28,11 +34,11 @@ const CategoryImage = (props) => {
...
-  width: '20em',
-  height: '20em'
-}
+// CategoryImage.defaultProps = {
+//   style: {},
+//   width: '20em',
+//   height: '20em'
+// }
 
 // connect go here???????
```
Comment (relyt4me) [inline code/web/src/modules/styleSurvey/CategoryImage.js:43]: Connect will only need to go there if we are using redux for that component

File: code/web/src/modules/styleSurvey/Survey.js [hunk 2]
```diff
@@ -44,13 +44,54 @@ class Survey extends Component {
...
             style={{
-              minWidth: '80vw'
+              width: 'auto'
             }}
           >
             <Category />
           </GridCell>
-         
-          
+          <GridCell
```
Comment (relyt4me) [inline code/web/src/modules/styleSurvey/Survey.js:52]: We will need to pass props into this with a map eventually but the format looks really good

File: code/web/src/modules/styleSurvey/category.js [hunk 3]
```diff
@@ -3,7 +3,8 @@ import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { Grid, GridCell } from '../../ui/grid'
 import { H3, H4 } from '../../ui/typography'
-import { grey, grey2, secondary } from '../../ui/common/colors'
+import { grey, grey2, grey3, secondary, white } from '../../ui/common/colors'
+import { level1, level2, level3, level4, level5 } from '../../ui/common/shadows'
```
Comment (relyt4me) [inline code/web/src/modules/styleSurvey/category.js:7]: I wrote a note go clean up imports for these later

**Review by relyt4me — APPROVED** [review]
This looks really good, still need to do some cleanup and actually connect a map but the layout is exactly right

File: code/web/src/modules/styleSurvey/CategoryImage.js [hunk 4]
```diff
@@ -1,22 +1,28 @@
...
 import PropTypes from 'prop-types'
+import { level1, level2, level3, level4, level5 } from '../../ui/common/shadows'
+import { grey, grey2, grey3, grey4, secondary, white } from '../../ui/common/colors'
 
 // Component
 const CategoryImage = (props) => {
   const { children, image, width, height, style, shadow, ...others } = props
-
+  // the conditional will have to be tied to state based on ifClicked
+  const boxBorder = (true === true) ? 'none' : `.7em solid ${grey4}`
```
Comment (Leasw144) [inline code/web/src/modules/styleSurvey/CategoryImage.js:11]: So given the comment, this trues in this ternary will eventually be tied to some aspect in the global state? Is that right?

**Comment by Leasw144** [comment]
This looks really great, good job on getting everything sorted

**Pull request merged**